<?xml version="1.0" ?>
<sdf version="1.6">
    <model name="grasp_joint">
        <joint name="gripper_box_joint" type="fixed">
            <parent>gripper_link</parent>
            <child>object_1::link</child>
            <pose>0 0 0.03 0 0 0</pose>
        </joint>
    </model>
</sdf>